spec/views/layouts/__snapshots__/application_by_election.snap

```diff
@@ -117,6 +117,11 @@
 </li>
 </ul>
 </div>
+<div class='container'>
+<p>
+Published and promoted by Tom de Grunwald on behalf of Swap My Vote, a campaign name of FORWARD DEMOCRACY LIMITED registered at 100 Church Street, Brighton, BN1 1UJ. Registered company in England & Wales no. 11707599.
```

Comment:
I thought this was illegal in HTML and that this should be `&amp;`, but I just looked it up and it's perfectly fine!  I wonder why GitHub's PR review UI highlights the `&` in black...